common/on_actions/eaw_on_actions.txt

```diff
@@ -2655,55 +2956,156 @@ on_actions = {
 			#End of EQC MARESOC Civil War
 			else_if = {
 				limit = {
-					FROM = {
-						original_tag = EQC
-						has_country_flag = is_maresoc
-					}
+					FROM = { original_tag = EQC }
 					original_tag = EQC
 				}
-				country_event = { id = mareland.26 days = 1 }
-			}
-			else_if = {
-				limit = {
-					FROM = {
-						original_tag = EQC
+				if = {
+					limit = {
+						FROM = { has_country_flag = is_maresoc }
 					}
-					original_tag = EQC
-					has_country_flag = is_maresoc
+					country_event = { id = mareland.26 days = 1 }
+				}
+				else_if = {
+					limit = {
+						has_country_flag = is_maresoc
+					}
+					country_event = { id = mareland.27 days = 1 }
 				}
-				country_event = { id = mareland.27 days = 1 }
 			}
 			else_if = {
 				limit = {
 					original_tag = LUS
-					FROM = { original_tag = LUS has_government = neutrality }
+					FROM = { has_government = neutrality }
 				}
 				country_event = { id = lushi_peasants.1 days = 50 random = 480 }#days to hours applied
 			}
 			else_if = {
 				limit = {
 					original_tag = FIR
-					FROM = { original_tag = FIR }
 				}
 				country_event = { id = FIR.29 days = 1 }
 			}
-		}
-	}
-	# ROOT is capitulated country, FROM is winner
-	on_capitulation = {
```

Comment:
No. No. No. Whatever removing this is for. No.

Reply:
Take a closer look, just because of the large number of changes before this, git incorrectly selects brackets, although in fact the entire effect was moved lower.